### PR TITLE
AdMob adapter throws exception on OnResume and OnPause. Vungle autorotation enabled.

### PR DESCRIPTION
**Pull request by mihailogazda**

### CODE REVIEW

**Comment by chauduyphanvu1** [comment]
@mihailogazda, thank you for the PR! We are fixing the AdMob `onResume()` and `onPause()` issue by [removing them](https://github.com/mopub/mopub-unity-mediation/issues/34#issuecomment-479126799) since they are no longer needed in the new AdMob rewarded video API.

Regarding the Vungle rotation change, could you elaborate on the context behind that change? 

Vu Chau
MoPub SDK Team

**Comment by mihailogazda** [comment]
Hey @chauduyphanvu1 Vungle one is an oldie, but from what i can see they haven't fixed it. More info here: https://github.com/mopub/mopub-android-mediation/pull/56

File: Vungle/src/main/java/com/mopub/mobileads/VungleInterstitial.java [hunk 1]
```diff
@@ -117,6 +117,9 @@ public void run() {
             Object ordinalViewCount = localExtras.get(ORDINAL_VIEW_COUNT_KEY);
             if (ordinalViewCount instanceof Integer)
                 mAdConfig.setOrdinal((Integer) ordinalViewCount);
+			
+			//	Set autorotation
+			mAdConfig.setAutoRotate(true);
```
Comment (chauduyphanvu1) [inline Vungle/src/main/java/com/mopub/mobileads/VungleInterstitial.java]: @ashinagawa, could you please help confirm this change? Is it a recommended fix from Vungle? If not, what is an alternative?

Comment (palrahul) [reply Vungle/src/main/java/com/mopub/mobileads/VungleInterstitial.java]: @chauduyphanvu1  we do not want this to be always set to true for adapter and instead take the auto-rotate as another local extra and then pass that to our SDK. We plan to submit a PR from Vungle by early next week.
@mihailogazda thanks for bringing this to our notice

**Comment by chauduyphanvu1** [comment]
Thanks for the review, @palrahul. Closing this PR in favor of the upcoming one.

**Comment by mihailogazda** [comment]
Cheers guys, I'll keep an eye for the updates.

**Pull request closed without merging**